Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -520,7 +520,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.airwallex.paymentacceptance;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match Development com.airwallex.paymentacceptance";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
Is this intentional?

Reply:
Yeah, sorry I forget to mention this in the description. I've changed the signing settings to use Automatic for Debug configuration builds and kept Manual for Release configuration. This should make development life easier while keeping CI/CD happy.